measure/performance/MSPerf.py

```diff
@@ -171,6 +171,18 @@ def main():
         printLog(output_log_handle, '#' * 69)
         printLog(output_log_handle, 'MSPERF (MULTI STREAMS PERFORMANCE) v0.20.06.03')
         printLog(output_log_handle, '#' * 69 + '\n')
+
+    if use_vdenc:
+        isSupported_vdenc_h264high = subprocess.Popen(['bash', '-c', './isVdenc.bash H264High'])
```

Comment:
Please, change permissions for isVdenc.bash:
```
chmod a+x isVdenc.bash
```

And drop `'bash', '-c', ` here.